### PR TITLE
ISPN-16802 Reduce javadoc warnings

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-16802

Pretty big PR with the intent to greatly reduce the Javadoc warnings our build generates.
Some warnings are impossible to remove (e.g. the use of Unsafe)

### CODE REVIEW

**Review by rigazilla — APPROVED** [review]

**Comment by tristantarrant** [comment]
Build is green

File: server/resp/src/main/java/org/infinispan/server/resp/commands/list/LLEN.java [hunk 1]
```diff
@@ -13,12 +13,12 @@
 import io.netty.channel.ChannelHandlerContext;
 
 /**
- * @link https://redis.io/commands/llen/
- *
+ * @link <a href="https://redis.io/commands/llen/">LLEN</a>
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/list/LLEN.java:16]:
```suggestion
 * <a href="https://redis.io/commands/llen/">LLEN</a>
```

File: server/resp/src/main/java/org/infinispan/server/resp/commands/sortedset/ZDIFF.java [hunk 2]
```diff
@@ -3,14 +3,14 @@
 import org.infinispan.server.resp.commands.sortedset.internal.DIFF;
 
 /**
+ * @see <a href="https://redis.io/commands/zdiff/">ZDIFF</a>
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/sortedset/ZDIFF.java:6]:
```suggestion
 * <a href="https://redis.io/commands/zdiff/">ZDIFF</a>
```

File: server/resp/src/main/java/org/infinispan/server/resp/commands/string/GETEX.java [hunk 3]
```diff
@@ -19,9 +19,7 @@
 import io.netty.channel.ChannelHandlerContext;
 
 /**
- * GETEX Resp Command
- *
- * @link https://redis.io/commands/getex/
+ * @link <a href="https://redis.io/commands/getex/">GETEX</a>
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/string/GETEX.java:22]:
```suggestion
 * <a href="https://redis.io/commands/getex/">GETEX</a>
```

File: server/resp/src/main/java/org/infinispan/server/resp/commands/string/SET.java [hunk 4]
```diff
@@ -15,7 +15,7 @@
 import io.netty.channel.ChannelHandlerContext;
 
 /**
- * @see <a href="https://redis.io/commands/set/">Redis documentation</a>
+ * @see <a href="https://redis.io/commands/set/">SET</a>
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/string/SET.java:18]:
```suggestion
 * <a href="https://redis.io/commands/set/">SET</a>
```

File: server/resp/src/main/java/org/infinispan/server/resp/commands/string/SETEX.java [hunk 5]
```diff
@@ -11,14 +11,15 @@
...
+ * <a href="https://redis.io/commands/setex/">SETEX</a>.
  * `<code>SETEX key seconds value</code>` command.
  * <p>
  * As of Redis 2.6.2, this command is deprecated. Applications should utilize `<code>SET key value EX seconds</code>`.
  * </p>
  *
  * @since 15.0
- * @author José Bolina
- * @see <a href="https://redis.io/commands/setex/">Redis documentation</a>.
+ * @author Jos&eacute; Bolina
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/string/SETEX.java:21]: Sorry @jabolina  :D

File: server/resp/src/main/java/org/infinispan/server/resp/commands/tx/MULTI.java [hunk 6]
```diff
@@ -34,7 +34,7 @@
  * and enter pub-sub mode.
  *
  * @since 15.0
- * @see <a href="https://redis.io/commands/multi/">Redis documentation</a>
+ * @see <a href="https://redis.io/commands/multi/">MULTI</a>
```
Comment (wburns) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/tx/MULTI.java:37]:
```suggestion
 * <a href="https://redis.io/commands/multi/">MULTI</a>
```

**Review by wburns — COMMENTED** [review]
Just some random little things, looks great !

**Comment by tristantarrant** [comment]
I have normalized all the resp commands using the following format:
```
CMD

@see <a href="https://redis.io/command/cmd/">CMD</a>
@since X.Y
```
I've also removed redundant docs, unless the command deviates from the redis implementation.

**Comment by wburns** [comment]
LGTM, restarted CI.

**Comment by wburns** [comment]
Integrated into main, thanks @tristantarrant !

**Pull request merged**